docs/en/administration/upgrade.md

```diff
@@ -19,6 +21,42 @@ For the file system that has been mounted using the old version of JuiceFS clien
 When unmounting the file system, make sure that no application is accessing it. Otherwise the unmount will fail. Do not forcibly unmount the file system, as it may cause the application unable to continue to access it as expected.
 :::
 
+### Smooth Upgrade
+
+Starting from version v1.2, JuiceFS supports the smooth upgrade feature, which allows you to mount JuiceFS again at the same mount point to achieve a seamless client upgrade. In addition, this feature can also be used to dynamically adjust mount parameters.
+
+Here are two common scenarios for illustration:
+
+1. Client Upgrade
+   For example, if there is a `juicefs mount` process `juicefs mount redis://127.0.0.1:6379/0 /mnt/jfs -d`, and you want to deploy a new JuiceFS client without unmounting the mount point, you can perform the following steps:
+
+   ```shell
+    # 1. Backup the current binary
+    cp juicefs juicefs.bak
+   
+    # 2. Download the new binary to overwrite the current juicefs binary
+   
+    # 3. Execute the juicefs mount command again to complete the smooth upgrade
+    juicefs mount redis://127.0.0.1:6379/0 /mnt/jfs -d
+    ```
+
+2. Dynamically Adjust Mount Parameters
+
+For example, if there is a `juicefs mount` process `juicefs mount redis://127.0.0.1:6379/0 /mnt/jfs -d`, and you want to adjust the log level to debug without unmounting the mount point, you can execute the following command:
+
+```shell
+# Adjust the log level
+juicefs mount redis://127.0.0.1:6379/0 /mnt/jfs --debug -d
+```
+
+Some notes:
+
+1. The smooth upgrade requires that the JuiceFS client versions of both the old and new processes are at least v1.2.
+
+2. The FUSE parameters in the new mount parameters should be consistent with the old mount parameters, otherwise the smooth upgrade will continue to overwrite the mount at the current mount point.
```

Comment:
```suggestion
- The FUSE parameters in the new mount parameters should be consistent with the old mount parameters, otherwise the smooth upgrade will overwrite the mount at the current mount point.
```